psydac/cad/tests/test_geometry.py

```diff
@@ -147,6 +147,7 @@ def test_geometry_2d_4():
     geo.export('circle.h5')
 
 #==============================================================================
+@pytest.mark.xfail # Changes in SymPDE make this fail
```

Comment:
Do you know if this will be fixed in branch `devel-mp-ddm`?

Reply:
The tests haven't been changed and they pass on Said's PR so I think that they will.